gcm.go

```diff
@@ -194,6 +195,12 @@ func (c *httpGcmClient) send(apiKey string, m HttpMessage) (*HttpResponse, error
 		return nil, fmt.Errorf("error sending request to HTTP connection server>%v", err)
 	}
 	gcmResp := &HttpResponse{}
+	gcmResp.Status = httpResp.StatusCode
+
+	if gcmResp.Status != http.StatusOK {
+		return gcmResp, fmt.Errorf("Encountered HTTP status code %d", gcmResp.Status)
```

Comment:
Fair point @bolshoy  - although from reading the reference ( https://developers.google.com/cloud-messaging/http-server-ref#error-codes ) it doesn't look like non-200 bodies contain anything particularly interesting. How would you suggest we return a raw body? Include a `RawBody []byte` in the `HttpResponse` struct?